l2lsp/methods/lsp_lifecycle.go

```diff
@@ -0,0 +1,88 @@
+// lsp_lifecycle.go
+package methods
+
+import (
+	"os"
+
+	"github.com/HexmosTech/lama2/l2lsp/lsp_req"
+	"github.com/HexmosTech/lama2/l2lsp/lsp_res"
+	"github.com/rs/zerolog/log"
+)
+
+func Initialize(request lsp_req.JSONRPCRequest) lsp_res.JSONRPCResponse {
+	/*
+		{
+			"jsonrpc": "2.0",
+			"id": 1,
+			"method": "initialize",
+			"params": {
+				"processId": null,
+				"clientInfo": {
+					"name": "MyEditor",
+					"version": "1.0.0"
+				},
+				"rootUri": "file:///path/to/workspace",
+				"capabilities": {
+					"workspace": {
+						"applyEdit": true,
+						"workspaceEdit": {
+							"documentChanges": true
+						},
+						"didChangeConfiguration": {
+							"dynamicRegistration": true
+						}
+					},
+					"textDocument": {
+						"publishDiagnostics": {
+							"relatedInformation": true
+						}
+					}
+				}
+			}
+		}
+	*/
```

Comment:
Why is this commented out?

Reply:
Is it some sort of structure to be expected?

Reply:
It is an example for the request, just so we can copy and put it in the terminal after starting the server to debug.